src/transferwise_clj/api/core.clj

```diff
@@ -0,0 +1,26 @@
+(ns transferwise-clj.api.core
+  (:require [transferwise-clj.constants :as const]
+            [transferwise-clj.data.core :as data]
+            [transferwise-clj.api.telegram.core :as telegram]
+            [transferwise-clj.api.transferwise.core :as transferwise]))
+
+(defn telegram-send-advise
+  "Send the quote advise for the user"
+  [savings actual-rate last-rate]
+  (telegram/send-msg
+   const/message-url
+   const/chat-id
+   (telegram/formatted-msg
+    const/message
+    (data/new-telegram-map
```

Comment:
I understand where you're coming from, but it is more idiomatic to use the map literals wherever you can so that there isn't a lot of indirection between the data structure and the function that operates on it.